core/src/test/scala/integration/kafka/api/ClientIdQuotaTest.scala

```diff
@@ -0,0 +1,62 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package kafka.api
+
+import java.util.Properties
+
+import kafka.admin.AdminUtils
+import kafka.server.{KafkaConfig, QuotaConfigOverride, QuotaId}
+import org.apache.kafka.common.security.auth.KafkaPrincipal
+import org.junit.Before
+
+class ClientIdQuotaTest extends BaseQuotaTest {
+
+  override def generateConfigs() = {
+    FixedPortTestUtils.createBrokerConfigs(serverCount,
```

Comment:
Is there a reason to use FixedPortTestUtils, which is discouraged since it can cause port conflicts?